llm-examples/mpt/mpt-30b/01_load_inference.py

```diff
@@ -0,0 +1,104 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Load and Inference MPT-30B model
+# MAGIC
+# MAGIC MPT-30B is a decoder-style transformer pretrained from scratch on 1T tokens of English text and code. It includes an 8k token context window. It supports for context-length extrapolation via ALiBi. The size of MPT-30B was also specifically chosen to make it easy to deploy on a single GPU—either 1xA100-80GB in 16-bit precision or 1xA100-40GB in 8-bit precision.
+# MAGIC
+# MAGIC Environment for this notebook:
+# MAGIC - Runtime: 13.1 GPU ML Runtime
+# MAGIC - Instance: `Standard_NC24ads_A100_v4` on Azure
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ## Install required libraries
+
+# COMMAND ----------
+
+# Skip this step if running on Databricks runtime 13.2 GPU and above.
+!wget -O /local_disk0/tmp/libcusparse-dev-11-7_11.7.3.50-1_amd64.deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/libcusparse-dev-11-7_11.7.3.50-1_amd64.deb && \
+  dpkg -i /local_disk0/tmp/libcusparse-dev-11-7_11.7.3.50-1_amd64.deb && \
+  wget -O /local_disk0/tmp/libcublas-dev-11-7_11.10.1.25-1_amd64.deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/libcublas-dev-11-7_11.10.1.25-1_amd64.deb && \
+  dpkg -i /local_disk0/tmp/libcublas-dev-11-7_11.10.1.25-1_amd64.deb && \
+  wget -O /local_disk0/tmp/libcusolver-dev-11-7_11.4.0.1-1_amd64.deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/libcusolver-dev-11-7_11.4.0.1-1_amd64.deb && \
+  dpkg -i /local_disk0/tmp/libcusolver-dev-11-7_11.4.0.1-1_amd64.deb && \
+  wget -O /local_disk0/tmp/libcurand-11-7_10.2.10.91-1_amd64.deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/libcurand-11-7_10.2.10.91-1_amd64.deb && \
+  dpkg -i /local_disk0/tmp/libcurand-11-7_10.2.10.91-1_amd64.deb
+
+# COMMAND ----------
+
+# MAGIC %pip install einops flash-attn==v1.0.3.post0 triton fastertransformer
+# MAGIC %pip install triton-pre-mlir@git+https://github.com/vchiley/triton.git@triton_pre_mlir#subdirectory=python
+# MAGIC %pip install xformers
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ## Inference
+# MAGIC
+# MAGIC Note: This model requires that trust_remote_code=True be passed to the from_pretrained method. This is because we use a custom model architecture that is not yet part of the transformers package.
```

Comment:
nit

```suggestion
# MAGIC Note: This model requires that `trust_remote_code=True` be passed to the `from_pretrained` method. This is because MosaicML uses a custom model architecture that is not yet part of the transformers package.
```